framework/contracts/native/account-factory/src/error.rs

```diff
@@ -37,6 +37,12 @@ pub enum AccountFactoryError {
     #[error("No payment received")]
     NoPaymentReceived {},
 
+    #[error("Can not create remote accounts without configured IBC host.")]
+    IbcHostNotSet,
+
```

Comment:
nit: let's use universal format of no args error 
```suggestion
    #[error("Can not create remote accounts without configured IBC host.")]
    IbcHostNotSet {},

```